mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java

```diff
@@ -84,6 +87,20 @@ public void initObjects() {
 
     }
 
+    public void initExpand() {
+        expandableListView.setOnGroupExpandListener(
+                new ExpandableListView.OnGroupExpandListener() {
+                    int prevExpandPos = -1;
```

Comment:
```suggestion
                    int previousExpandPosition = -1;
```
Keep variable names as descriptive as you can yet short.

Reply:
Okay!